ChangeLog.rst

```diff
@@ -1,3 +1,88 @@
+2017-04-27 azure 2.0.0rc7
++++++++++++++++++++++++++
+
+**Disclaimer**
+
+Starting with this release, we fixed several packaging issues to improve our package generation system
+- Uninstalling of packages using "pip uninstall" now work properly (#1150)
+- setuptools install of "azure" works properly (#728)
+
+Update from 2.0.0rc6 *might* work, but we strongly suggest to create a fresh virtual environment.
+
+**Features**
+
+***CLI configuration***
+
+You can now execute your SDK script using the configuration of the `AzureCLI <https://github.com/Azure/azure-cli>`__
+
+Example:
+
+    from azure.common.client_factory import get_client_from_cli_profile
+    from azure.mgmt.compute import ComputeManagementClient
+    client = get_client_from_cli_profile(ComputeManagementClient)
+    client.virtual_machines.get('rg', 'vm')
+
+Configuration from the CLI is:
```

Comment:
Note sure, I want people to get what is the impact on the SDK and then how to configure it using CLI